PULL_REQUEST_TEMPLATE

```diff
@@ -1,24 +1,32 @@
 Jira: [JIRA-TOKEN](https://ticketevolution.atlassian.net/browse/JIRA-TOKEN)
 
+> [!IMPORTANT]
+> Toggle to SQUASH AND MERGE when merging PRs to `staging` branch
+
 ## Why / What
+Why: These changes are for...
+
+What: This PR adds/removes/fixes... 
 
-This PR adds/removes/fixes&hellip;
+If this PR requires other developers to make changes to their local environment after this PR has been merged, please
+describe here. Examples: DB migration, Docker container restart/rebuild, .env changes etc.
 
 ## Testing
 
-I tested this code by&hellip;
+I tested this code by (unit/integration/manual) testing...
+
+## Environment Variables
+
+If there are any changes to environment variables, list here and link any relevant `tevo-kubernetes` PRs. Update the 
+project's .env file in 1Password after PR has been merged if necessary.
 
-## Reviewer Checklist (To be edited and marked by approver)
+## Database
 
-After approving the PR, the reviewer should check the boxes below. See the [Reviewer Checklist](https://ticketevolution.atlassian.net/wiki/spaces/EN/pages/3015016552/Reviewer+Checklist) company standard for more info on these bullet points.
+If any DB changes, describe here. Add DBAs to the PR reviewers to approve DB migrations
```

Comment:
Why do we think this is needed? A DB change would generally have a structure change, schema change, or migration file that makes that obvious no? Is this just a helper section for @ticketevolution/dba ?

Reply:
Essentially, but good to see at a glance before peeking into the structure/schema, or migration file changes

Reply:
If we're adding words to the template that we're specifically intending to be replaced with proper content, can that text be a different color?

Reply:
Color is hard: https://github.com/orgs/community/discussions/31570

Reply:
I wanted fewer words :( doh!